src/components/HomePage/Forms/MovieForm.tsx

```diff
@@ -1,32 +1,77 @@
-import React, { FC } from 'react';
+import React, { FC, FormEvent } from 'react';
+import { useDispatch } from 'react-redux';
 import { Form } from 'reactstrap';
-import { MovieInfo } from '../../../types';
+import { addMovieCreator, updateMovieCreator } from '../../../redux/actions/actionCreators/actionCreators';
+import { MovieInfoDetails } from '../../../types';
 import { ResetButton, SubmitButton } from '../../shared';
 import './Form.scss';
 import { InputWithLabel } from './InputWithLabel';
 import { SelectInputGenre } from './SelectInputGenre';
 
-export const MovieForm: FC<{ movieInfo?: MovieInfo }> = ({ movieInfo }) => {
+export const MovieForm: FC<{ movieInfo?: MovieInfoDetails }> = ({ movieInfo }) => {
+  const dispatch = useDispatch();
+
+  const submit = (e: FormEvent<HTMLElement>) => {
+    e.preventDefault();
+    // todo check if mandatory fields filled
+    const title = (document.getElementById('movieTitle') as HTMLTextAreaElement).value;
```

Comment:
Anyway we do not need to use js selectors to get values:

const [title, setTitle] = useState('')
...
<input onChange={event => setTitle(event.target.value)} />

Reply:
fixed in commit e9860952613603eb2cf3095f28c180d207b9668c